src/modules/0.0.14/aws_lambda/entity/lambda_function.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  AfterInsert,
+  AfterLoad,
+  AfterUpdate,
+  Column,
+  Entity,
+  JoinColumn,
+  ManyToOne,
+  PrimaryColumn,
+} from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+import { Role } from '../../aws_iam/entity';
+
+export enum Architecture {
+  arm64 = "arm64",
+  x86_64 = "x86_64"
+}
+
+export enum PackageType {
+  Image = "Image",
+  Zip = "Zip"
+}
+
+export enum Runtime {
+  dotnet6 = "dotnet6",
+  dotnetcore10 = "dotnetcore1.0",
+  dotnetcore20 = "dotnetcore2.0",
+  dotnetcore21 = "dotnetcore2.1",
+  dotnetcore31 = "dotnetcore3.1",
+  go1x = "go1.x",
+  java11 = "java11",
+  java8 = "java8",
+  java8al2 = "java8.al2",
+  nodejs = "nodejs",
+  nodejs10x = "nodejs10.x",
+  nodejs12x = "nodejs12.x",
+  nodejs14x = "nodejs14.x",
+  nodejs16x = "nodejs16.x",
+  nodejs43 = "nodejs4.3",
+  nodejs43edge = "nodejs4.3-edge",
+  nodejs610 = "nodejs6.10",
+  nodejs810 = "nodejs8.10",
+  provided = "provided",
+  providedal2 = "provided.al2",
+  python27 = "python2.7",
+  python36 = "python3.6",
+  python37 = "python3.7",
+  python38 = "python3.8",
+  python39 = "python3.9",
+  ruby25 = "ruby2.5",
+  ruby27 = "ruby2.7"
+}
```

Comment:
This changes relatively frequently. Can we query it into a table instead?

Reply:
I'm also surprised at how far behind Java is on this list. The newest here is Java 11, but [Java 18](https://en.wikipedia.org/wiki/Java_version_history) has been out for ~3 months.

Reply:
this is an enum itself in aws sdk, I can check if theres an api call that we can do to get this info

Reply:
I've done a quick search in the API and couldn't find an endpoint to retrieve this information. Either we can maintain this enum or directly use the one provided by AWS SDK? I did not use the last one to be consistent with other modules.